src/queue/dune

```diff
@@ -4,26 +4,31 @@
 (alias
  (name default)
  (package multicoretests)
- (deps lin_tests.exe))
+ (deps lin_tests_dsl.exe))
 
 (env
  (_
   (binaries
    (../check_error_count.exe as check_error_count))))
 
 
+(executable
+ (name lin_tests_dsl)
+ (modules lin_tests_dsl)
+ (flags (:standard -w -27))
+ (libraries qcheck lin))
+
 (executable
  (name lin_tests)
  (modules lin_tests)
  (flags (:standard -w -27))
- (libraries qcheck lin)
- (preprocess (pps ppx_deriving_qcheck ppx_deriving.show ppx_deriving.eq)))
+ (libraries qcheck lin))
 
 (rule
  (alias runtest)
  (package multicoretests)
- (deps lin_tests.exe)
+ (deps lin_tests_dsl.exe)
  (action
   (progn
-   (bash "(./lin_tests.exe --no-colors --verbose || echo 'test run triggered an error') | tee lin-output.txt")
+   (bash "(./lin_tests_dsl.exe --no-colors --verbose || echo 'test run triggered an error') | tee lin-output.txt")
    (run %{bin:check_error_count} "queue/lin_tests" 1 lin-output.txt))))
```

Comment:
To get a nice error message in case of failure, the string should also be updated here:
```
   (run %{bin:check_error_count} "queue/lin_tests_dsl" 1 lin-output.txt))))
```